generator/README.md

```diff
@@ -71,6 +71,18 @@ modules:
       # with that value.
       - old_index: bsnDot11EssIndex
         new_index: bsnDot11EssSsid
+     
+     overrides: # Allows for per-module overrides of bits of MIBs
+       metricName:
+         regexp_extract:
+           Temp: # A new metric will be created appending this to the metricName to become metricNameTemp.
+             - regex: '.*' # Regex to extract a value from the returned SNMP walks's value.
```

Comment:
For the example to make sense, this should be (.*) so that there is a $1

Also mention it's a float64 value